.github/workflows/slycot-build-and-test.yml

```diff
@@ -283,7 +283,9 @@ jobs:
           pip install slycot-wheels/${{ matrix.packagekey }}/slycot*.whl
           pip show slycot
       - name: Slycot and python-control tests
-        run: bash slycot-src/.github/scripts/run-tests.sh
+        run: JOBNAME=$JOBNAME bash slycot-src/.github/scripts/run-tests.sh
```

Comment:
I am surprised that this works without quotes around `$JOBNAME` as there is a space in the value.